code-generators/java-types/schema-artefacts/src/main/java/uk/gov/di/model/schema/VocabSchemaLoader.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.di.model.schema;
+
+import java.io.InputStream;
+
+public class VocabSchemaLoader {
```

Comment:
May be useful to have some predefined constants such as, so that you don't need to "know" the filename which could change:
```java
public static final String IDENTITY_CHECK_CREDENTIAL_JWT_NAME = "IdentityCheckCredentialJWT.json";
public static final String INHERITED_IDENTITY_JWT_NAME = "InheritedIdentityJWT.json";
public static final String RISK_ASSESSMENT_CREDENTIAL_JWT_NAME = "RiskAssessmentCredentialJWT.json";
public static final String SECURITY_CHECK_CREDENTIAL_JWT_NAME = "SecurityCheckCredentialJWT.json";
```